scaling_performance/network_optimization.adoc

```diff
@@ -0,0 +1,142 @@
+[[scaling-performance-routing-network-optimization]]
+= Routing and Network Optimization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+
+[[scaling-performance-scaling-router-haproxy]]
+== Scaling {product-title} HAProxy Router
+
+The {product-title}
+xref:../install_config/router/index.adoc#install-config-router-overview[router]
+is the ingress point for all external traffic destined for {product-title}
+services.
+
+On an Amazon EC2 m4.xlarge instance, a single HAProxy router is able to handle
```

Comment:
Please change as per reviewed: https://docs.google.com/document/d/1A_Lzym4s7Jg-f2O11VGAijr4IeNPrpnq5QkKlxGGKiI/edit

I.e. "On an public cloud instance of size 4 vCPU/16GB RAM instance" ...